src/iperf_api.c

```diff
@@ -2160,6 +2162,8 @@ get_parameters(struct iperf_test *test)
 	    iperf_set_test_udp_counters_64bit(test, 1);
 	if ((j_p = cJSON_GetObjectItem(j, "repeating_payload")) != NULL)
 	    test->repeating_payload = 1;
+	if ((j_p = cJSON_GetObjectItem(j, "zerocopy")) != NULL)
+	    test->zerocopy = 1;
```

Comment:
Thanks for the PR! Have not tried running this code yet, but perhaps this should be `test->zerocopy = j_p->valueint`?

Reply:
Changed to `test->zerocopy = j_p->valueint` as suggested and tested that it works o.k.

Reply:
Great, thanks! I have tested this also and it seems to be working correctly.